cmd/agent/subcommands/dogstatsdstats/command.go

```diff
@@ -110,7 +110,7 @@ func requestDogstatsdStats(log log.Component, config config.Component, cliParams
 	} else if cliParams.jsonStatus {
 		s = string(r)
 	} else {
-		s, e = dogstatsdServer.FormatDebugStats(r)
+		s, e = serverDebug.FormatDebugStats(r)
```

Comment:
I think it would be great to not export `FormatDebugStats`. Can it be part of the component interface?